.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
+  - '5.5'
```

Comment:
Oh, ok then my bad. just made them as the first example on the Travis documentation.

Sometimes if you see as a beginner different then in the official documentation may be confusing.